goty-server/src/test/kotlin/com/aleinin/goty/game/GameControllerTest.kt

```diff
@@ -0,0 +1,81 @@
+package com.aleinin.goty.game
+
+import com.api.igdb.request.IGDBWrapper
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.kotlin.any
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+internal class GameControllerTest {
+
+    lateinit var mockMvc: MockMvc
+
+    @Mock
+    lateinit var wrapper: IGDBWrapper
+
+    @Autowired
+    lateinit var objectMapper: ObjectMapper
+
+    @BeforeEach
+    fun setup() {
+        val client = IGDBGameSearchClient(wrapper, objectMapper)
+        val service = GameSearchService(client)
+        mockMvc = standaloneSetup(GameController(service)).build()
+    }
+
+    @Test
+    fun `Should return games`() {
+        val mockAPIResponse = """
+            [
+                {
+                    "id": 0,
+                    "name": "a game"
+                },
+                {
+                    "id": 1,
+                    "name": "a game II"
+                }
+            ]
+        """.trimIndent()
+        val expected = listOf(
+            GameSearchResponse("0", "a game"),
+            GameSearchResponse("1", "a game II")
+        )
+        Mockito.`when`(wrapper.apiJsonRequest(any(), any())).thenReturn(mockAPIResponse)
```

Comment:
Mockito Kotlin adds `whenever` as an alias for `when` so you can avoid the backticks.

Reply:
Is this in a later version than we're using? 

![image](https://user-images.githubusercontent.com/25621595/197433330-a8574822-eac6-4b59-bb7c-002c7acd589e.png)

Reply:
Nice! Resolvedi in [174485e](https://github.com/aleinin/game-of-the-year/pull/44/commits/174485e0d3f83ff80764156b7bae22dc1cc110cf)